### ISSUE TITLE
Hey!

### ISSUE THREAD

**Issue by nitinthewiz** [flechnical/halfscreen#1]
Hi there! What does this extension do?? Just curious...

**Comment by flechnical** [comment]
Hi Nitin, If you click on the extension icon it will restrict any fullscreen video you play (inside the tab you are on) to the size of the browser window. So for example, click on a YouTube video, then click on the extension icon and then make the video fullscreen and you will see that the video does not fill your entire monitor.

This effect happens because the extension fakes capturing the media inside the tab (it doesn't actually do anything with the media, it just asks the browser for access to it). For some reason Chrome won't allow a video to go fullscreen, when the tab it is in, is being "captured".

I got the idea from watching a streamer who uses the "Volume Master" Chrome extension. Everytime he uses that extension his videos aren't fullscreen anymore and I was curious why. So I looked at the Chrome extension dev-guide and figured out that that extension uses the tabCapture API to manipulate and boost the audio of a tab.
